jdbc/src/main/java/org/apache/zeppelin/jdbc/JDBCInterpreter.java

```diff
@@ -702,6 +702,13 @@ private InterpreterResult executeSql(String propertyKey, String sql,
               String results = getResults(resultSet,
                   !containsIgnoreCase(sqlToExecute, EXPLAIN_PREDICATE));
               interpreterResult.add(results);
+              try {
+                if (resultSet.isClosed()) {
+                  break;
```

Comment:
`if (resultSet.isClosed())` is true, we break out of the loop to run the sql statement.
is that an error condition there - since not everything in the paragraph is executed?